setup.cfg

```diff
@@ -3,6 +3,7 @@ ignore = W503, E203, B305
 max-line-length = 88
 
 [mypy]
+python_version = 3.6
```

Comment:
This makes sure mypy checks for compatibility with Python 3.6 and onwards, which seems to be the oldest version Starlette supports (?).